src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrForwardFlower.cs

```diff
@@ -394,25 +398,55 @@ private async Task CommentIncludedPRs(
         var result = await sourceRepo.ExecuteGitCommand(["log", "--pretty=%s", $"{lastCommit}..{currentCommit}"], cancellationToken);
         result.ThrowIfFailed($"Failed to get the list of commits between {lastCommit} and {currentCommit} in {sourceRepo.Path}");
 
-        (Regex regex, string prLinkFormat) = gitRepoType switch
+        (Regex regex, Regex? alternativeRegex, string prLinkFormat) = gitRepoType switch
```

Comment:
Tbh it would be better to return `Regex[]`

Reply:
true, less code duplication